composer.json

```diff
@@ -20,8 +20,6 @@
         "doctrine/common": "~2.4",
         "twig/twig": "~1.23|~2.0",
         "psr/log": "~1.0",
-        "symfony/security-acl": "~2.7",
```

Comment:
Merge without commit, then you can change the staged changes and commit (still a merge commit).